.github/workflows/test.yml

```diff
@@ -120,15 +116,12 @@ jobs:
     - name: Check out main branch code
       if: github.event_name != 'pull_request_target'
       uses: actions/checkout@v3
-      with:
-        fetch-depth: 2
 
     - name: Check out PR branch code
       if: github.event_name == 'pull_request_target'
       uses: actions/checkout@v3
       with:
         ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
did you check if this affects coverage?

Reply:
Added "test_cmd" to the Test step so amd64 runs cover/test-web, and arm64 does test-go/test-web instead.